### PR TITLE
LPD-17467 Importing Object - Custom State Flow doesn't resemble JSON file

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPD-17467](https://liferay.atlassian.net/browse/LPD-17467)

### CODE REVIEW

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java [hunk 1]
```diff
@@ -89,11 +89,20 @@ public class ObjectFieldSettingUtil {
 		if (serviceBuilderObjectFieldSetting.compareName(
 				ObjectFieldSettingConstants.NAME_STATE_FLOW)) {
 
+			ObjectStateFlow objectStateFlow = null;
+
+			if (!(objectFieldSetting.getValue() instanceof ObjectStateFlow)) {
```
Comment (p-albuquerque) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java:94]: In the case of this bug, the execution enter into the IF clause, right?
Do you know which case the execution enter in the ELSE?

**Review by p-albuquerque — CHANGES_REQUESTED** [review]
Just some questions

**Review by p-albuquerque — APPROVED** [review]

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java (hunk 1 above)
Comment (diogomelo09) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java:94]: when it passes the second PUT of the import. objectDefinitionResource.putObjectDefinition(
putObjectDefinition.getId(), putObjectDefinition);

**Comment by guilhermedcamacho** [comment]
Hey @diogomelo09 , can you please check if this [PR](https://github.com/liferay-objects/liferay-portal/pull/3162) already fixes this bug?

**Comment by diogomelo09** [comment]
> Hey @diogomelo09 , can you please check if this [PR](https://github.com/liferay-objects/liferay-portal/pull/3162) already fixes this bug?

Hey @guilhermedcamacho, I tested it and it doesn't work.

**Comment by guilhermedcamacho** [comment]
Hey @diogomelo09 , sorry for the delay. I arranged with @linolaoj for him to take over this PR.

**Pull request closed without merging**